src/components/play/layer-settings/LayerSettings.js

```diff
@@ -545,13 +586,16 @@ class LayerSettings extends Component {
             showInstrumentsList,
             showArticulationOptions,
             showLayerPopup,
+            showHamburgerPopup,
             selectedInstrument,
-            showVolumePopup
+            showVolumePopup,
+            windowWidth
         } = this.state;
 
         const { classes, user } = this.props
         const selectedLayer = this.props.selectedLayer
         const userColors = this.getUserColors()
+        const isMobile = windowWidth <= 480
```

Comment:
Sorta surprised 480 isn't a setting somewhere, ideally this would be global and a var

Reply:
Should be I guess, I'll look into it.

Reply:
you're 100% right, I think it was a temp hack that turned into a component wide soln, on it ASAP.